jenkins-pipelines/perf-simple-query-i4i.jenkinsfile

```diff
@@ -0,0 +1,12 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    region: 'eu-west-1',
+    test_name: 'microbenchmarking_test.PerfSimpleQueryTest.test_perf_simple_query',
+    test_config: 'test-cases/microbenchmarking/amazon_perf_simple_query_i4i.yaml',
```

Comment:
We don't need this entire yaml duplicated, only a configuration yaml with one parameter that was changed.

Reply:
changed, have to change [sdcm/sct_config.py](https://github.com/scylladb/scylla-cluster-tests/pull/6512/files#diff-4964b8575600051f35ba15bc41a6cdd5576162b0882a61eea12b3013091c67ab) little as well